docs/buildah-build.1.md

```diff
@@ -327,9 +336,10 @@ Run up to N concurrent stages in parallel.  If the number of jobs is greater tha
 stdin will be read from /dev/null.  If 0 is specified, then there is
 no limit on the number of jobs that run in parallel.
 
-**--label** *label*
+**--label** *label[=value]*
 
 Add an image *label* (e.g. label=*value*) to the image metadata. Can be used multiple times.
+If *label* is named, but no `=` or `value` is provided, then the *label* is removed from the image.
```

Comment:
```suggestion
If *label* is named, but neither `=` or `value` is provided, then the *label* is removed from the image.
```

Reply:
Going to neither/nor for the lot of these.